compose/apps/firefox.yml

```diff
@@ -1,20 +1,28 @@
+#########################
+# firefox.yml
+#########################
+#
+# This container runs Firefox
+
 services:
   # ####################
-  # An example of running an unprivileged X11 app in this environment. It's
-  # also an example of running a 3rd-party container that wasn't necessarily
-  # designed with GoW in mind.
+  # An example of running an unprivileged X11 app in this environment.
   firefox:
-    image: andrewmackrodt/firefox-x11
+    image: ghcr.io/games-on-whales/firefox:edge
+#   build:
+#     context: ./images/firefox
+#     args:
+#       BASE_IMAGE: ${BUILD_BASE_IMAGE}
+#       BASE_APP_IMAGE: ${BUILD_BASE_APP_IMAGE}
+    runtime: ${DOCKER_RUNTIME}
     volumes:
       - ${XORG_SOCKET}:/tmp/.X11-unix
-      # Pulse socket, audio
       - ${PULSE_SOCKET_HOST}:${PULSE_SOCKET_GUEST}
-      # If you have permission errors make sure to chomd ${id}:${id} ./local_state
-      - ${local_state}/ffox:/run/user/1000
 
     env_file:
       - config/common.env
       - config/xorg.env
+      # run-gow: gpu_env
```

Comment:
What does this do?

Reply:
it's a hint to the `run-gow` script that it needs to insert some GPU-specific stuff in there at runtime. we do it this way (simple text substitution) to avoid depending on a tool like `yq` that actually parses the yaml

Reply:
Ah fair, do you mind just adding a one liner here saying that too? It would be nice to have that note here in the future if someone tries to use this as a template to add a new app.